src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,106 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INIT_SIZE = 10;
+    private static final double RESIZE_LIST = 1.5;
+    private static final int FIRST_ELEMENT = 0;
+    private int size;
+    private T[] elementsData = (T[]) new Object[INIT_SIZE];
```

Comment:
Common mistakes: try to use constractor  

public ArrayList() {
        this.elements = (T[]) new Object[DEFAULT_CAPACITY];
    }